compiler/src/iree/compiler/Codegen/SPIRV/test/BUILD.bazel

```diff
@@ -16,6 +16,11 @@ package(
 
 iree_lit_test_suite(
     name = "lit",
+    # Several of these tests are somewhat expensive with sanitizers. Tagging
+    # them large does not currently disable them on sanitizers, as we
+    # currently run sanitizers on all but "enormous" tests, but is done
+    # for consistency with similarly expensive tests (#14152).
+    size = "large",
```

Comment:
This is suspicious. I don't think any tests under `compiler/` should be tagged "large" and I also don't think we should set test sizes based purely on some specific configuration (e.g. ASan on Linux) being slow.

Reply:
ok, dropping these changes.